src/main/java/org/sagebionetworks/web/client/widget/entity/menu/v3/DefaultActionConfigurationUtil.java

```diff
@@ -0,0 +1,169 @@
+package org.sagebionetworks.web.client.widget.entity.menu.v3;
+
+import java.util.Arrays;
+import java.util.EnumMap;
+import java.util.List;
+import java.util.Map;
+import org.sagebionetworks.web.client.jsinterop.entity.actionmenu.ActionConfiguration;
+import org.sagebionetworks.web.client.widget.entity.menu.v2.Action;
+
+public class DefaultActionConfigurationUtil {
+
+  private DefaultActionConfigurationUtil() {
+    super();
+  }
+
+  private static Map<Action, ActionConfiguration> createMap() {
```

Comment:
The existing implementation has the analog to this in widget XML, but since we aren't using that for the widget's view. It's now in code.